Modules/gcmodule.c

```diff
@@ -1405,6 +1411,40 @@ gc_is_tracked(PyObject *module, PyObject *obj)
     return result;
 }
 
+/*[clinic input]
+gc.freeze
+
+Freeze all current tracked objects and ignore them for future collections.
+
+This can be used before a fork to make the gc copy-on-write friendly.
```

Comment:
Rather than "a fork", perhaps use the wording "a POSIX fork() call", as that may not be obvious to all readers.